src/blueprint/operators/const_field.rs

```diff
@@ -25,27 +28,120 @@ fn validate_data_with_schema(
 pub struct CompileConst<'a> {
     pub config_module: &'a config::ConfigModule,
     pub field: &'a config::Field,
-    pub value: &'a serde_json::Value,
+    pub value: &'a ValueOrDynamic,
     pub validate: bool,
 }
 
+#[derive(Debug, Clone)]
+pub enum MustacheOrValue {
+    Mustache(Mustache),
+    Value(serde_json::Value),
+}
+
+impl MustacheOrValue {
+    pub fn is_const(&self) -> bool {
+        match self {
+            MustacheOrValue::Mustache(m) => m.is_const(),
+            _ => true,
+        }
+    }
+}
+
+#[derive(Debug, Clone)]
+pub enum ValueOrDynamic {
```

Comment:
Can be called `DynamicValue` and move to it's own file.

Reply:
Done